portfolio/src/main/webapp/script.js

```diff
@@ -15,7 +15,7 @@
 /**
  * Adds a random greeting to the page.
```

Comment:
Very minor thing, if you change the method name, you should change the comment to reflect the changes.  So here you should say it adds a random fact instead of a random greeting.